src/components/login/facebook-login/facebook-login.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import PropTypes from "prop-types";
+import FacebookLogin from "react-facebook-login";
+import "../google-facebook-login.scss";
+import { FaFacebook } from "react-icons/fa";
+import ButtonNames from "../../../constants/ButtonNamesConstants";
+
+const { FACEBOOK_LOGIN } = ButtonNames;
+
+const LoginFacebook = ({ config }, props) => {
```

Comment:
Why we have two separate parameters here, there should be only one instead. Can you please verify if everything is correct here

Reply:
Changed